src/main/java/create_task_use_case/CreateTaskPresenter.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Create Task Presenter Class.
+ * Author: Fardin Faruk
+ * Modified By: N/A
+ * Created: Nov 19, 2022
+ * Last Modified: N/A
+ */
+package create_task_use_case;
+
+public class CreateTaskPresenter implements CreateTaskOutputBoundary {
+    @Override
+    public CreateTaskOutputData prepareSuccessView(CreateTaskOutputData response) {
+        //View Model Update
+        response.setSuccessfullyCreated(true);
+        return response;
+    }
+
+    @Override
+    public CreateTaskOutputData prepareFailView(String error) {
+        //View Model Update
+        //throw new CreateTaskFailed(error);
+        return null;
```

Comment:
This should have some implementation before View Models are added. This method should still return outputData with successfullyCreated set to false and containing an error message corresponding to the parameter error.

Reply:
Updated output Boundary Structure to reflect saving error messages and successfullyCreated set to false.